src/gui/DetailsWidget.cpp

```diff
@@ -263,23 +264,21 @@ void DetailsWidget::getSelectedGroup(Group* selectedGroup)
 
 void DetailsWidget::updateTotp()
 {
-    if (m_locked) {
-        m_timer->stop();
-        return;
+    if (!m_locked) {
+        QString totpCode = m_currentEntry->totp();
+        QString firstHalf = totpCode.left(totpCode.size()/2);
```

Comment:
@adolfogc can you add spaces around the operators please?

Reply:
Sure.